torch/nn/quantized/functional.py

```diff
@@ -0,0 +1,5 @@
+from torch._ops import ops
+from torch._C import _add_docstr
```

Comment:
do we have plans to add doc strings?

Reply:
I tried, looks like we can't, c10 ops already have doc string.

Reply:
what does that mean in practice?  Will these functions have docstrings?

Reply:
cc @smessmer